packages/schematics/src/collection/node-application/files/app/proxy.conf.json

```diff
@@ -0,0 +1,10 @@
+{
+  "/graphql": {
```

Comment:
Should this be added as a default?

Reply:
I'm on the fence about it. On one side, anyone using graphql will need something like this. On the other hand the default app doesn't need it

Reply:
Yeah I feel like we should keep it out because the default app doesn't need it.

We could introduce `--framework nest:graphql` to create the best experience for starting a GraphQL project.